### ISSUE TITLE
Wrong shift time bug

### ISSUE THREAD

**Issue by lzk228** [space-wizards/space-station-14#17131]
## Description

Sometime PDA can display 8 or 15 hours when really it was only 2-3 hours
Also time in end round menu display wrong
That can be checked with discord stat

**Reproduction**

**Screenshots**

**Additional context**

**Comment by lzk228** [comment]
isnt happening in a while so ill close it

**Comment by deltanedas** [comment]
this still happens im pretty sure

**Comment by metalgearsloth** [comment]
I think if you disconnect and rejoin it's cooked or somethin.

**Comment by deltanedas** [comment]
yeah when latejoining its the same time as on the lobby but sometimes when relogging it becomes wrong

**Comment by lzk228** [comment]
oh ok
